src/gui/MovieImportDialog.cpp

```diff
@@ -529,6 +529,11 @@ void MyMovieImportDialog::ImportClick( wxCommandEvent& event )
 		// for adding to the database..
 		main_frame->current_project.database.BeginMovieAssetInsert();
 
+		// list of added movies for subsequent metadata insertion
+		// 
+		wxArrayInt list_asset_id;
```

Comment:
The advantage of wxArray is that it will compile everywhere, no matter if and what version of STL is available. However, it seems to me like nowadays that should not be an issue anymore. @timothygrant80 do you think STL would be ok?

Reply:
It's only a minor preference, and since wxArray is essentially a wrapper to std::vector anyway, maybe it is better to leave it as is, especially in the gui code. I didn't think about it impacting compiling. Though I will point out, the GPU code requires standard 17

Reply:
it is true that by now std::vector should work on anything we care about.  At some point we could change all the arrays to vectors, but I think we should be consistent.